dbtmetabase/__init__.py

```diff
@@ -289,6 +311,13 @@ def shared_opts(func: Callable) -> Callable:
         multiple=True,
         help="Additional HTTP request header to be sent to Metabase.",
     )
+    @click.option(
+        "--metabase_requests_timeout",
+        cls=OptionAcceptableFromConfig,
+        type=INT_OR_NONE,
```

Comment:
There should never be a none value. If you personally don't need one, you can set it to say 24 hours, but there should always be some value.

Reply:
i have no idea why i read in the issue that it could be None, fixing it